shell/browser/ui/electron_web_modal_dialog_manager_delegate.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SHELL_BROWSER_UI_CHROME_WEB_MODAL_DIALOG_MANAGER_DELEGATE_H_
+#define SHELL_BROWSER_UI_CHROME_WEB_MODAL_DIALOG_MANAGER_DELEGATE_H_
```

Comment:
The `#ifndef` name should be the same as the path, so in this case it would be `SHELL_BROWSER_UI_ELECTRON_WEB_MODAL_DIALOG_MANAGER_DELEGATE_H_`.